<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #a4bb7b;</string>
  </property>
  <widget class="QFrame" name="frameSettings">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>800</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #c4e09b; border-radius: 14px;</string>
   </property>
   <widget class="QLabel" name="labelReserve">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>388</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>color:#222;</string>
    </property>
    <property name="text">
     <string>Показывать количество </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkReserve">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QCheckBox::indicator { width: 180px; height: 80px; }
QCheckBox::indicator:checked { image: url(:/icon/on.png); }
QCheckBox::indicator:unchecked { image: url(:/icon/off.png); }
         </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelSound">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>381</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>color:#222;</string>
    </property>
    <property name="text">
     <string>Звук</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkSound">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QCheckBox::indicator { width: 180px; height: 80px; }
QCheckBox::indicator:checked { image: url(:/icon/on.png); }
QCheckBox::indicator:unchecked { image: url(:/icon/off.png); }
         </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelHighlight">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>539</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>color:#222;</string>
    </property>
    <property name="text">
     <string>Подсветка подходящих домино</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkHighlight">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>230</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QCheckBox::indicator { width: 180px; height: 80px; }
QCheckBox::indicator:checked { image: url(:/icon/on.png); }
QCheckBox::indicator:unchecked { image: url(:/icon/off.png); }
         </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPlayers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Количество игроков</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinPlayers">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelBots">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Количество ботов</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBots">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>440</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelReserve_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>color:#222;</string>
    </property>
    <property name="text">
     <string>домино в резерве</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHighlight_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>539</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>color:#222;</string>
    </property>
    <property name="text">
     <string>домино</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>489</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="buttonLayout"/>
   </widget>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#222;  font-size: 38px
</string>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background: transparent;</string>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/home.png</normaloff>:/icon/home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
